src/css.js

```diff
@@ -113,6 +113,18 @@ function getWidthOrHeight( elem, name, extra ) {
 		styles = getStyles( elem ),
 		isBorderBox = jQuery.css( elem, "boxSizing", false, styles ) === "border-box";
 
+	// Support: IE11+
+	// Fix for edge case in IE 11. See https://github.com/jquery/jquery/issues/1764
+	if ( !support.reliableFullscreenBox && window.top !== window.self &&
+		document.msFullscreenElement ) {
```

Comment:
@mzgol 

> Anyway, any such change is a plan reaching far further than this PR as it'd require much larger effort

For the whole `jQuery.support` yes, but we discussing here encapsulation of this specific method.

@gibson042 Well here, we need to change some things, like not `parent` but `window.parent`